### PR TITLE
[16.0][MIG] sale_report_salesperson_from_partner: Migration to 16.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT51908

@carlosdauden @sergio-teruel please review

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

File: sale_report_salesperson_from_partner/tests/test_sale_report_salesperson_from_partner.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        # from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT
+        DISABLED_MAIL_CONTEXT = {
+            "tracking_disable": True,
+            "mail_create_nolog": True,
+            "mail_create_nosubscribe": True,
+            "mail_notrack": True,
+            "no_reset_password": True,
+        }
+        cls.env = cls.env(context=dict(cls.env.context, **DISABLED_MAIL_CONTEXT))
+        cls.salesperson = new_test_user(
```
Comment (carlos-lopez-tecnativa) [inline sale_report_salesperson_from_partner/tests/test_sale_report_salesperson_from_partner.py]: Could you inherit from [BaseCommon](https://github.com/odoo/odoo/blob/8f0b8d451f97dd7d0f118b541a80e9674af51124/odoo/addons/base/tests/common.py#L19)

**Review by carlos-lopez-tecnativa — CHANGES_REQUESTED** [review]
LGTM just a minor change ;)

File: sale_report_salesperson_from_partner/tests/test_sale_report_salesperson_from_partner.py (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply sale_report_salesperson_from_partner/tests/test_sale_report_salesperson_from_partner.py]: Right, thanks

File: sale_report_salesperson_from_partner/report/sale_order.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    user_from_partner_id = fields.Many2one(
```
Comment (sergio-teruel) [inline sale_report_salesperson_from_partner/report/sale_order.py]: Is this related needed??

**Review by sergio-teruel — CHANGES_REQUESTED** [review]
Minor comment

File: sale_report_salesperson_from_partner/report/sale_order.py (hunk 2 above)
Comment (pilarvargas-tecnativa) [reply sale_report_salesperson_from_partner/report/sale_order.py]: Yes, to be able to add the filters to the orders and quotations in the ‘sale.order’ views as it is already done with the user_id.

https://github.com/OCA/sale-reporting/pull/296/files#diff-68c46ca14dab463b454ba14f74ba68c1310b07da9c17650a2d6fe45706696c0dR11

**Comment by pilarvargas-tecnativa** [comment]
@sergio-teruel Removed also in migration

**Comment by sergio-teruel** [comment]
Fix tests... Thanks

File: sale_report_salesperson_from_partner/tests/test_sale_report_salesperson_from_partner.py [hunk 3]
```diff
@@ -1,22 +1,15 @@
...
-        # Remove this variable in v16 and put instead:
-        # from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT
-        DISABLED_MAIL_CONTEXT = {
-            "tracking_disable": True,
-            "mail_create_nolog": True,
-            "mail_create_nosubscribe": True,
-            "mail_notrack": True,
-            "no_reset_password": True,
-        }
         cls.env = cls.env(context=dict(cls.env.context, **DISABLED_MAIL_CONTEXT))
```
Comment (rousseldenis) [inline sale_report_salesperson_from_partner/tests/test_sale_report_salesperson_from_partner.py]: You could inherit from `BaseCommon` test class to avoid this line

Comment (pilarvargas-tecnativa) [reply sale_report_salesperson_from_partner/tests/test_sale_report_salesperson_from_partner.py]: Done! thanks

**Comment by pilarvargas-tecnativa** [comment]
> Fix tests... Thanks

Solved!

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor
